buildbot/configure.py

```diff
@@ -276,6 +280,7 @@ def main():
     parser.add_argument("--disable-preview-lib", action='store_true', help="Disable building of the SYCL runtime major release preview library")
     parser.add_argument("--disable-fusion", action="store_true", help="Disable the kernel fusion JIT compiler")
     parser.add_argument("--add_security_flags", type=str, choices=['none', 'default', 'sanitize'], default=None, help="Enables security flags for compile & link. Two values are supported: 'default' and 'sanitize'. 'Sanitize' option is an extension of 'default' set.")
+    parser.add_argument('--native_cpu_libclc_targets', help='Target triples for libclc, used by the Native CPU backend')
```

Comment:
Seems like hyphens are the norm here, except for `add_security_flags`. Thus I'd suggest `native-cpu-libclc-targets`

Reply:
We should probably also rename `add_security_flags` for the users' sake.

Reply:
I went for `native-cpu-libclc-targets`, thank you. (I'd rather not rename `add_security_flags` as part of this PR)

Reply:
Yeah, renaming that flag isn't for this PR :+1: